examples/remoteBuild.md

```diff
@@ -1,18 +1,36 @@
 # remote targets
 
-`kustomize build` can be run on a URL.
+`kustomize build` can be run on a URL. Resources can also reference other
+kustomization directories via URLs too.
 
-A [lite version of go-getter module](https://github.com/yujunz/go-getter) is
-leveraged to get remote content, then running `kustomize build` against the
-desired directory in the local copy.
+The URL format is any valid `git clone` URL with an optional directory and some
```

Comment:
This also precludes simple `github.com/owner/repo` style URLs but I think overall it's much clearer to say `git clone` URL instead. It's much easier for users too, since if it works with `git clone` then it surely works for kustomize.

The same can't be said for all the other "go-getter" style URLs. You could use the `go-getter` CLI to test but not everyone has that available and our format isn't even 100% compatible with it.

Reply:
WDYT about adding a quick comment to this paragraph:
`Kustomize also supports github.com/owner/repo style  URLs such as github.com/kubernetes-sigs/kustomize/examples/multibases/dev/?ref=v1.0.6` ?

Reply:
Those types of urls fall under the "legacy formats". I believe we should no longer officially support said URLs as they're confusing.

Points of confusion,

 * Bare github.com works but not for any new hosts like git.sr.ht
 * Does github.com mean https or ssh?
 * The special github handling supports using `/` instead of `//` to indicate the directory

I could be convinced to keep them but I think it just adds unnecessary complexity for both users and the code that handles urls.